README.md

```diff
@@ -840,6 +840,7 @@ Table of Contents
   * [Substack](https://substack.com) — Unlimited free newsletter service. Start paying when you charge for it.
   * [Tempmailo](https://tempmailo.com/) - Unlimited free temp email addresses. Autoexpire in two days.
   * [HotTempMail](https://hottempmail.com/) - Unlimited free temp email or disposable temporary email addresses. Autoexpires in one day.
+   * [gettempemail.com](https://gettempemail.com) — A disposable email service is a service that allows users to generate a temporary email address that can be used for a short period of time.
```

Comment:
These services with no ownership details or any TOS or anything is just risky, who knows what happens with these emails.

So I will reject this one and audit the existing ones on the list, if they are similarly sketchy they will be removed.


Thanks

Reply:
can I create PR again after adding TOS or ownership details ?

Reply:
Of course